### DISCUSSION TITLE
Is it possible to have the panel group extend the viewport?

### DISCUSSION THREAD

**Discussion by ReidCampbell** [bvaughn/react-resizable-panels#328] [Q&A] [answered]
I'd like to be able to set a minSize of a Panel and if I try to increase the size of a sibling Panel where it would usually stop due to not enough room it would overflow and allow me to scroll to view the other panel. Is this possible?

**Comment by bvaughn** [comment, marked as answer]
Hmm. No, I don't think the use case you describe sounds like something that makes for this library. Thanks for understanding 🙇🏼
